<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetTestItem</class>
 <widget class="QWidget" name="WidgetTestItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>152</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>288</width>
    <height>152</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>288</width>
    <height>152</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
color: rgb(255, 255, 255);
font: 18px &quot;.Helvetica Neue DeskInterface&quot;;
}
QGroupBox {
	border-width:1px;
	border-style:solid;
	border-color:lightGray;
	margin-top: 0.5ex;
	border-radius:10px
}
QGroupBox::title {
	
	color: rgb(85, 170, 255);
   subcontrol-origin: margin;
   subcontrol-position: top left;
   left:10px;
   margin-left: 0px;
   padding:0 1px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="title">
      <string>治具SN</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QStackedWidget" name="stackedWidget">
        <widget class="QWidget" name="page_index">
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>串口号:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_port">
            <property name="text">
             <string>UNKNOW</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>设备SN:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_deviceSN">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>UNKNOW</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="openExternalLinks">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>设备号:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_device">
            <property name="text">
             <string>UNKNOW</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="1">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_free">
         <property name="styleSheet">
          <string notr="true">font: 24px &quot;.Helvetica Neue DeskInterface&quot;;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>78</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>无治具载入</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>78</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_testing">
         <property name="styleSheet">
          <string notr="true">font: 24px &quot;.Helvetica Neue DeskInterface&quot;;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>24</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="button_more">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{
	border-top-left-radius: 4px;
	border-top-right-radius: 4px;
	background-color: rgb(255, 255, 255);
	color: rgb(66, 66, 66);
	font: 18px &quot;.Helvetica Neue DeskInterface&quot;;
}
QPushButton:pressed{
background-color: rgba(255, 255, 255, 128);
}
</string>
              </property>
              <property name="text">
               <string>正在检测...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>5</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>5</height>
               </size>
              </property>
              <property name="contextMenuPolicy">
               <enum>Qt::NoContextMenu</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QProgressBar {
    border: 0px solid #76797C;
    border-radius:1px;
    text-align: center;
	background-color: transparent;
}

QProgressBar::chunk {
    background-color:rgb(85, 170, 255);
	width: 20px;
	height:5px;
}</string>
              </property>
              <property name="value">
               <number>24</number>
              </property>
              <property name="textVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>24</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_result">
         <widget class="QPushButton" name="pushButton_4">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>30</y>
            <width>141</width>
            <height>91</height>
           </rect>
          </property>
          <property name="text">
           <string>成功</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
